src/engine/shared/entities/player.lua

```diff
@@ -75,7 +86,7 @@ function player:player()
 	self:setCollisionBounds( min, max )
 
 	self:networkNumber( "id", player.lastPlayerId + 1 )
-	self:networkNumber( "moveSpeed", 2 )
+	self:networkNumber( "moveSpeed", 4 )

```

Comment:
Declined, run speed in `vertex-adventure` (default) is this, and yet to be implemented